highfive/configs/rust-lang/rust.json

```diff
@@ -23,14 +23,19 @@
             "compiler-team-contributors"
         ],
         "libs": [
-            "@dtolnay",
```

Comment:
Is this change intentional? I don't see it in the discussion I think?

Reply:
Yea, the split was david's idea for helping improve the libs-api teams review bandwidth by making it so libs-api focused reviewers, using himself as an example, don't have to triage and review non API changes and can focus more time on API changing PRs.